scripts/cargo.sh

```diff
@@ -18,19 +31,20 @@ echo "\tGXI Plugindir: ${GREEN}${GXI_PLUGIN_DIR}${NO_COLOR}
 \tBuild profile:       ${GREEN}${GXI_BUILD_PROFILE}${NO_COLOR}
 "
 
-cd $1
+cd "$1"
 if [ "$6" = "true" ]; then
     path="${4}/${RUST_TARGET}/release/${5}"
 else
     path="${4}/release/${5}"
 fi
+
 if [ "$7" = "true" ]; then
     features="--features gtk_v3_22"
 fi
 
 if [ "$GXI_BUILD_PROFILE" = "development" ]; then
-	path=$(echo $path | sed 's|release|debug|')
-	cargo build --target-dir ${4} ${features} && cp "${path}" ${2}/${3}
+	path="$(echo $path | sed 's|release|debug|')"
+	cargo build --target-dir "${4}" "${features}" && cp "${path}" "${2}/${3}"
```

Comment:
The quotes from features have to go for this to work, otherwise it interprets the '--features gtk_v3_22' as one argument.

Reply:
done